framework/BaseForm.js

```diff
@@ -0,0 +1,18 @@
+import Label from '../elements/Label.js'
+
+class BaseForm {
+  constructor(name, selector) {
+    this.name = name
+    this.selector = selector
+    this.label = new Label(this.selector, this.name)
+  }
+
+  async isDisplayed(timeout) {
+    return await this.label.isDisplayed({ timeout })
```

Comment:
please fix this issue in other methods as well. you don't need to use await when you are returning something. if your method is marked as async it will return promise in any case even if in code you return non-promise value